src/Roadkill.Web/App_Data/NLog.config

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<nlog xmlns="http://www.nlog-project.org/schemas/NLog.xsd" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+	<targets>
+		<target name="FileTarget" xsi:type="File"
+			layout="[${longdate:universalTime=True}] [${level}] ${message}"
```

Comment:
Yes it is,  pass the exception as first parameter. See docs with examples: https://github.com/NLog/NLog/wiki/How-to-log-exceptions

(and maybe use `. Log() ` and drop the switch?)